packages/stellar_solana_bridge/src/components/BridgeSolana.vue

```diff
@@ -1,24 +1,41 @@
 <template>
-  <div class="d-flex flex-column" style="min-height: 100vh">
+  <div
+    class="d-flex flex-column"
+    style="min-height: 100vh"
+  >
     <v-alert
       transition="fade-transition"
-      v-model="alert"
       border="start"
       type="info"
       close-label="Close Alert"
       color="primary"
       variant="tonal"
-      closable
       class="mb-0"
-      style="max-height: 64px"
+      :max-height="$vuetify.display.xs ? 80 : 64"
```

Comment:
the recommended way to check the display in vuetify with vue 3 is:  

```ts
import { useDisplay } from 'vuetify'

const { xs } = useDisplay()

```

https://vuetifyjs.com/en/features/display-and-platform/#usage